public/voice-processor.js

```
@@ -2,14 +2,34 @@
+    this.buffer = new Float32Array(this.bufferSize);
+    this.bufferIndex = 0;
   }
 
   process(inputs) {
     // inputs: [ [ Float32Array, ... ] ]
     const input = inputs[0];
     if (input && input[0]) {
-      // 將 Float32Array 轉成 transferable buffer 傳回主執行緒
-      this.port.postMessage(input[0]);
+      const inputData = input[0];
+      
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)